lib/services/pushNotifications.dart

```diff
@@ -9,17 +8,15 @@ class PushNotificationService {
   final NavigationService? _navigationService = locator<NavigationService>();
 
   Future init() async {
-    if (Platform.isIOS) {
-      await _fcm.requestPermission(
-        alert: true,
-        announcement: false,
-        badge: true,
-        carPlay: false,
-        criticalAlert: false,
-        provisional: false,
-        sound: true,
-      );
-    }
+    await _fcm.requestPermission(
```

Comment:
I think that we should check if Android version is equal or above 13 since only then the permission is needed

Reply:
Im guessing on older version of android they will just return that permissions are granted (without doing a popup thing) but im not sure. I should probably setup an emulator for an older version of android to find out...